components/frontend/src/report/IssueTracker.js

```diff
@@ -36,29 +37,34 @@ export function IssueTracker({ report, reload }) {
     const [issueEpicFieldSupported, setIssueEpicFieldSupported] = useState(false) // Does the current project and issue type support epic links?
     useEffect(() => {
         let didCancel = false
-        get_report_issue_tracker_options(report.report_uuid).then(function (json) {
-            if (!didCancel) {
-                // For projects, use the project key as value to store because that's what users entered when this wasn't a single choice option yet
-                setProjectOptions(json.projects.map(({ key, name }) => ({ key: key, value: key, text: name })))
-                setProjectValid(json.projects.some(({ key }) => key === report.issue_tracker?.parameters?.project_key))
-                // For issue types, use the name as value to store because that's what users entered when this wasn't a single choice option yet
-                setIssueTypeOptions(
-                    json.issue_types.map(({ key, name }) => ({
-                        key: key,
-                        value: name,
-                        text: name,
-                    })),
-                )
-                setIssueTypeValid(
-                    json.issue_types.some(({ name }) => name === report.issue_tracker?.parameters?.issue_type),
-                )
-                setIssueEpicOptions(json.epic_links.map(({ key, name }) => ({ key: key, value: key, text: name })))
-                const fieldKeys = json.fields.map((field) => field.key)
-                setLabelFieldSupported(fieldKeys.includes("labels"))
-                const fieldNames = json.fields.map((field) => field.name.toLowerCase())
-                setIssueEpicFieldSupported(fieldNames.includes("epic link"))
-            }
-        })
+        get_report_issue_tracker_options(report.report_uuid)
+            .then(function (json) {
+                if (!didCancel) {
+                    // For projects, use the project key as value to store because that's what users entered when this wasn't a single choice option yet
+                    setProjectOptions(json.projects.map(({ key, name }) => ({ key: key, value: key, text: name })))
+                    setProjectValid(
+                        json.projects.some(({ key }) => key === report.issue_tracker?.parameters?.project_key),
```

Comment:
Interesting that it's not a problem to have long comment line length

Reply:
Prettier is nog 100% strict about the max line length.